penpost/penpost-web
# Gaj/feature/success message

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 
Added a "SuccessMessage" component to be displayed once a user logs in to the app and goes through the postcard process. Once they hit send postcard a success message will appear letting them know their postcard was successfully sent. The message then has a button to hit OK and routes back home.

Also fixed a bug with the login component to ensure a user cannot login unless the username/password are longer than 6 characters.

Issues closed #

### Problems Encountered & Resulting Solutions  
> State any problems encountered 
N/A


> Describe your code changes in detail for reviewers.
Added Success Component to be displayed in the Postcard preview
changes "username" state from username to email
fixed a bug in login component to ensure a user cannot login until the character length is 6 letters or longer


### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?
App, Login, SuccessComponent


### How should this be tested?
> What steps need to be taken to confirm functionality? 
npm start in browser window


### Steps To Take
> Any questions and possible steps moving forward.
